src/main/java/online/talkandtravel/controller/http/ChatController.java

```diff
@@ -41,34 +48,72 @@ public class ChatController {
 
   private final ChatService chatService;
 
+  /**
+   * updates id of last read message of chat by user
+   * @param dtoRequest userId and lastReadMessageId
+   */
+  @PostMapping("/{chatId}/messages/set-last-read-message")
```

Comment:
According to naming convention of REST principals, we should not use verbs that describe action. Action is described in request Type (in this case it is POST).
So, please, remove word `set`.
2nd. We already have word `messages` in url, we don't need to type it one more time. So remove `message`.
After this you will have `"/{chatId}/messages/last-read"`